docs/collab/133.md

```diff
@@ -1,4 +1,28 @@
 # 5	Weitere Open-Science-Themen
 ## 5.1 Berücksichtigung von Open Source in Förderinstrumenten
 
-[Format: Stand heute – Problem – Vorschlag einer Massnahme]
+Im Rahmen der in den letzten zwei Jahren etablierten [Community of Practice "Open Source"](https://bfh.github.io/contact#bfhchopensource) haben wir aus Sicht (AHB, HAFL, TI, W und Vizerektorat) erste strategische [Anlaufpunkte zum Thema Open Source](http://bfh.ch/opensource) an der BFH aufbauen können.
+
+Analog zu 4.1 stellen auch wir fest, dass die "Mitarbeit an" und "Freigabe von" quelloffenen Systemen bisher nicht als eigenständige (und gleichwertige) Leistung in der Forschungsinitiative berücksichtigt wird (s. [#52](https://github.com/bfh/opensource/issues/52)).
+
+Nebst einem **klaren Committment zur präferierten ([open-by-default](https://en.wikipedia.org/wiki/Open_by_default)) [Nutzung (1), Mitarbeit (2) & Freigabe (3)](https://bfh.github.io) von Artefakten** unter freien und offenen [Lizenzen nach SPDX](https://spdx.org/licenses/): 
+   - für Software: [Open Source Initiative (OSI)](https://opensource.org/license),
+   - für Dokumentation & kreative Inhalte: [Creative Commons (CC)](https://creativecommons.org/cclicenses/),
+   - sowie für (offenen) Daten, KIs und Hardware
+
+in öffentlichen, frei zugänglichen, nicht kommerziellen Repositorien (wie z.B. [github.com/bfh](https://github.com/bfh)), schlagen wir vor explizit zu Fördern und zu Honorieren
+ - (1) Open-Source-Artefakte zu **Nutzen** u.a. 
```

Comment:
Als 1. Punkt würde ich hinzufügen: 
(1) Valorisation von Open-Source-Publikationen 
   - systematische Erfassung der Open-Source-Publikationen in ARBOR
   - Open Source Publikationen werden wie Peer-Reviewed Publikationen behandelt und erhalten diesselben Incentives/Valorisation

Vergleiche Issue https://github.com/bfh/opensource/issues/52, Samuel Pfyffer ist hier ja involviert - evtl. ist dieser Antrag auch genau der von ihm erwähnte Vorstoss in diese Richtung, entsprechend sollte dieser Punkt hier auf jeden Fall auftauchen.